distributor-api-v1/operations.md

```diff
@@ -1135,8 +1140,81 @@ In case of an error caused by insufficient availability \(which might have decre
 | `Client` | string | required | Identification of the client as described in [Authorization](https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/authorization). |
 | `HotelId` | string | required | Unique identifier of the hotel. |
 | `ReservationGroupId` | string | required | Unique identifier of the reservation group. |
+| `Extent` | Object of [Extent](operations.md#reservation-group-get-extent) | optional | Object specifying whether to return related data |
+
+### Extent <a id="reservation-group-get-extent"></a>
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `PaymentRequests` | boolean | optional | Whether the response should contain PaymentRequests related to the reservation group |
+| `Payments` | boolean | optional | Whether the response should contain Payment attempts related to the PaymentRequest for reservation group |
+
 
 ### Response <a id="response-7"></a>
 
-Same as in [Create Reservation Group](operations.md#create-reservation-group).
+```json
+{
+    "Id": "f6fa7e62-eb22-4176-bc49-e521d0524dee",
+    "CustomerId": "7ac6ca0d-7c08-4ab1-8da8-9b44979d8855",
+    "Reservations": [],
+    "TotalAmount": { },
+    "PaymentRequestId": "2e3a700a-7b10-4e61-8e9f-acfa00ee00df",
+    "PaymentRequests": [],
+    "Payments": []
+}
+```
+
+This part provide documentation only for the related data in `PaymentRequests` & `Payments` rest of the response is the
```

Comment:
PaymentRequestId description was missing. I will add it to the `Create Reservation Group` (in table bellow [this example](https://github.com/MewsSystems/gitbook-distributor-guide/pull/48/files#diff-e0d8177d43ad27101a34daf5ee1404fa5b293c8884f331abcbe852ac6671a819R1086))

Mainly I do not want to describe the whole response here again. Basically it will be the same, but creation of `ReservationGroup` does not allow the extent so I would like to show and document the extra fields provided by the Extent

Let's discuss this in person, you have way more experiance with docs from Connector, so I am open to any suggestion 👍